crates/swc_ecma_transforms_module/tests/fixture/common/issue-4988/2/input.ts

```diff
@@ -0,0 +1,3 @@
+import * as testUtils from "vs/base/test/common/testUtils";
+
+export import flakySuite = testUtils.flakySuite;
```

Comment:
I think we should return/export something for this syntax

Reply:
OK, It should be handled by TS strip pass.

Reply:
This issue is about the test config.
There is no TS strip pass in the swc_ecma_transforms_module tests.

In actual use, the strip pass of TS is always  before the module pass, so there is no such issue.
I should move this test case to swc integration test.